spec/features/gis_accessioning_spec.rb

```diff
@@ -62,8 +62,12 @@
                                                   workflow: 'gisAssemblyWF',
                                                   workflow_retry_text: 'Error: extract-boundingbox',
                                                   retry_wait: 5) do |page|
-      click_link_or_button 'Reindex'
+      page.driver.with_playwright_page do |page|
```

Comment:
There seems to be a known issue (https://github.com/YusukeIwaki/capybara-playwright-driver/issues/83) where playwright just doesn't find something in the DOM. This line consistently raised a `not found in DOM` error when testing. The "workaround" described in the linked issue does work, which might lend itself to adding a helper method if there are a sufficient number of these situations.